sql/moz-fx-data-shared-prod/firefox_ios_derived/attributable_clients_v1/query.sql

```diff
@@ -3,23 +3,52 @@ RETURNS INT64 AS (
   (SELECT SUM(value) FROM UNNEST(map))
 );
 
-WITH client_search_activity AS (
+WITH client_days AS (
   SELECT
     DATE(submission_timestamp) AS submission_date,
     sample_id,
     client_info.client_id,
+    LOGICAL_OR(mozfun.norm.extract_version(app_version, 'major') >= 108) AS has_search_data,
     SUM(sum_map_values(metrics.labeled_counter.search_in_content)) AS searches,
     SUM(sum_map_values(metrics.labeled_counter.browser_search_with_ads)) AS searches_with_ads,
     SUM(sum_map_values(metrics.labeled_counter.browser_search_ad_clicks)) AS ad_clicks,
   FROM
     firefox_ios.baseline
   WHERE
-    DATE(submission_timestamp) = @submission_date
+    {% if is_init() %}
+      submission_date >= "2020-05-01"
```

Comment:
Why May and not January?

Reply:
Data started being reported for `firefox_ios.baseline` in May. I can add a link to a query